### DISCUSSION TITLE
Welcome to vscode-jupyter-toc Discussions!

### DISCUSSION THREAD

**Discussion by xelad0m** [xelad0m/vscode-jupyter-toc#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ztingray** [comment]
Great add-on which helps me structure large notebooks. Thanks for your generously shared efforts!
However, I would like to change style on headers (Fonts, sizes, etc) on different levels but fail to find where I can edit these setting. Any hint would be appreciated.
ray

**Reply by xelad0m** [reply]
Thank you for your assessment

Most aspects of the style in VS Code are defined by extensions related to themes. And you can also manually set style for individual headers in markdown cell, for example

```
# <p style="font-family: 'Times New Roman'; font-size: 28px; color: Green;">Header1</p>

Text...

## <p style="font-family: 'Arial'; font-size: 18px; color: Red;">Header2</p>

Text...
```

I'm not sure that there is an easy way to set such style settings for jupyter notebooks headers in general case.

Good luck
